libs/blocks/notification/notification.css

```diff
@@ -48,8 +49,7 @@
   font-weight: unset;
 }
 
-.notification [class*="heading-"] + p,
-.notification.pill p {
+.notification .text p:not(.icon-area, .action-area) {
```

Comment:
Such selectors usually increase specificity, making it a bit more difficult to overwrite if needed. Is there any other way to apply the styles with less specific selectors?